package.json

```diff
@@ -48,7 +48,7 @@
     "ts-jest": "~29.1",
     "ts-node": "^10.8.0",
     "turbo": "^1.9.9",
-    "typescript": "^4.6.4"
```

Comment:
ahhh, all the sweeeeeet wins

this avoids a whole host of issues with next-13, see: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-1.html#decoupled-type-checking-between-jsx-elements-and-jsx-tag-types, don't know if we hit them or not but good to know